src/lib/ui/thermostat/TemperatureControl.tsx

```diff
@@ -0,0 +1,124 @@
+import { type ChangeEventHandler, useEffect, useRef } from 'react'
+
+import { TemperatureAddIcon } from 'lib/icons/TemperatureAdd.js'
+import { TemperatureSubtractIcon } from 'lib/icons/TemperatureSubtract.js'
+
+const DEFAULT_MIN = 50
+const DEFAULT_MAX = 90
+
+interface TemperatureControlProps {
+  variant: 'heat' | 'cool'
+  value: number
+  onChange: (temperature: number) => void
+  min?: number
+  max?: number
+  unit?: 'fahrenheit' | 'celsius'
+}
+
+export function TemperatureControl({
+  variant,
+  value,
+  onChange,
+  min = DEFAULT_MIN,
+  max = DEFAULT_MAX,
```

Comment:
I don't think we need to shared consts anymore.

```suggestion
interface TemperatureControlProps {
  variant: 'heat' | 'cool'
  value: number
  onChange: (temperature: number) => void
  min?: number
  max?: number
  unit?: 'fahrenheit' | 'celsius'
}

export function TemperatureControl({
  variant,
  value,
  onChange,
  min = 50,
  max = 90,
```